lib/src/layer.dart

```diff
@@ -56,13 +57,14 @@ class Layer {
       assembleTileMatrix(inflatedString);
     }
 
-    properties = TileMapParser._parsePropertiesFromElement(element);
+    properties = TileMapParser.parsePropertiesFromElement(element);
   }
 
   // TMX data format documented here: https://github.com/bjorn/tiled/wiki/TMX-Map-Format#data
   void assembleTileMatrix(var bytes) {
-    tileMatrix = List.generate(height, (_) => List<int>(width));
-    tileFlips = List.generate(height, (_) => List<Flips>(width));
+    tileMatrix = List.generate(height, (_) => List<int>.filled(width, 0));
+    tileFlips = List.generate(height,
+        (_) => List<Flips>.filled(width, const Flips(false, false, false)));
```

Comment:
Not part of this PR, but these generate functions should be able to fill the lists with the correct values directly instead of having the loops afterwards.